packages/wonder-blocks-core/src/components/render-state-root.js

```diff
@@ -49,9 +47,10 @@ export const RenderStateRoot = ({
     );
 };
 
-// Flow doesn't know about defaultProps on functional components like this,
-// and it doesn't seem worth the effort to teach it.
-// $FlowIgnore[prop-missing]
+// We can set `defaultProps` on a functional component if we move the `export` to appear
+// afterwards.
 RenderStateRoot.defaultProps = {
     throwIfNested: true,
 };
+
+export {RenderStateRoot};
```

Comment:
This is unrelated to the other changes, but I thought I may as well fix this since I was in the neighborhood.